src/Function/Relation/Binary/Equality.agda

```diff
@@ -0,0 +1,35 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Function Equality setoid
+------------------------------------------------------------------------
+
+{-# OPTIONS --cubical-compatible --safe #-}
+
+module Function.Relation.Binary.Equality where
+
+  open import Function.Bundles using (Func; _⟨$⟩_)
+  open import Level using (Level)
+  open import Relation.Binary.Bundles using (Setoid)
+
+  private
+    variable
+      a₁ a₂ b₁ b₂ c₁ c₂ : Level
+      
+  setoid : Setoid a₁ a₂ → Setoid b₁ b₂ → Setoid _ _
+  setoid From To = record
+    { Carrier = Func From To
+    ; _≈_ = λ f g → ∀ {x} → f ⟨$⟩ x To.≈ g ⟨$⟩ x
+    ; isEquivalence = record
+      { refl = To.refl
```

Comment:
And the proofs of reflexivity, symmetry, transitivity can be lifted out as their own lemmas.

Reply:
All done - though it's not quite as nice since I have to eta-expand.